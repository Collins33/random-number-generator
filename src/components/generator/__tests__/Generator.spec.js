import React from "react";
import { shallow } from "enzyme";
import Generator from "../generator";

describe("Generator form component", () => {
  it("renders correctly", () => {
    const generatorComponent = shallow(<Generator />);
    expect(generatorComponent).toMatchSnapshot();
  });
});
