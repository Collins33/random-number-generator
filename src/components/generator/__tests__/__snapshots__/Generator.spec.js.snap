// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Generator form component renders correctly 1`] = `ShallowWrapper {}`;
